mono/io-layer/processes.h

```diff
@@ -10,6 +10,7 @@
 #ifndef _WAPI_PROCESSES_H_
 #define _WAPI_PROCESSES_H_
 
+#include <sys/types.h>
```

Comment:
We do not guard sys/types.h anywhere else in io-layer: https://github.com/mono/mono/search?p=2&q=%22sys%2Ftypes.h%22+path%3Amono%2Fio-layer&type=Code&utf8=%E2%9C%93